src/shared/plugins/error/index.js

```diff
@@ -27,28 +25,29 @@ const isUnauthorized = request => {
 
 const _handler = async (request, h) => {
   const res = request.response;
+  const { pluginOptions } = h.realm;
 
   if (isIgnored(request) || !res.isBoom || is404(request)) {
     return h.continue;
   }
 
   // Destroy session for CSRF error
   if (isCsrfError(request)) {
-    logger.info(pick(res, ['error', 'message', 'statusCode', 'stack']));
+    pluginOptions.logger.info(pick(res, ['error', 'message', 'statusCode', 'stack']));
     return request.logOut();
   }
 
-  // Unauthorised - redirect to sign in
+  // Unauthorised - redirect to welcome
   if (isUnauthorized(request)) {
-    logger.info(pick(res, ['error', 'message', 'statusCode', 'stack']));
-    return h.redirect('/signin');
+    pluginOptions.logger.info(pick(res, ['error', 'message', 'statusCode', 'stack']));
```

Comment:
I think it may be neater if this piece of logic moved to a handler in the auth plugin, as then there is no dependency between the plugins